pre_processing/netcdf_output_create_file.F90

```diff
@@ -154,9 +154,7 @@ subroutine netcdf_create_rtm(global_atts, source_atts, cyear, cmonth, cday, chou
 
    ! Set number of vertical levels/layers here, as GFS is different to ECMWF
    kdim = preproc_dims%kdim+1
-   if (nwp_flag .eq. 6) kdim = kdim-1
-   if (nwp_flag .eq. 7) kdim = kdim-1
-   if (nwp_flag .eq. 8) kdim = kdim-1
+   if (nwp_flag .eq. 0) kdim = kdim-1
```

Comment:
Wouldn't it be easier to read
if (nwp_flag .eq. 0) then kdim = preproc_dims%kdim else kdim = preproc_dims%kdim+1